NightscoutUploadKit/NightscoutEntry.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NightscoutEntry.swift
+//  RileyLink
+//
+//  Created by Timothy Mecklem on 11/5/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+import MinimedKit
+
+public class NightscoutEntry : DictionaryRepresentable {
+    
+    public enum GlucoseType: String {
+        case Meter
+        case Sensor
+    }
+    
+    public let timestamp: Date
+    let glucose: Int
+    let previousSGV: Int?
+    let previousSGVNotActive: Bool?
+    let direction : String?
```

Comment:
(Personal note for Tim: using this for reference going forward https://github.com/raywenderlich/swift-style-guide#spacing)